libkbfs/folder_update_prepper.go

```diff
@@ -24,6 +25,9 @@ type folderUpdatePrepper struct {
 	folderBranch FolderBranch
 	blocks       *folderBlockOps
 	log          logger.Logger
+
+	cacheLock   sync.Mutex
```

Comment:
As `go vet` has identified, all the methods of `folderUpdatePrepper` now need to be accessed by pointer.